crates/db_views/src/post_view.rs

```diff
@@ -100,17 +99,6 @@ fn queries<'a>() -> Queries<
     ),
   );
 
-  let is_saved = |person_id| {
-    post_saved::table
-      .filter(
-        post_aggregates::post_id
-          .eq(post_saved::post_id)
-          .and(post_saved::person_id.eq(person_id)),
-      )
-      .select(post_saved::published.nullable())
-      .single_value()
```

Comment:
I tried various things, but this was the culprit.

When I changed this subquery from a select to `exists`, like the `is_read` function just below it, the query is instant. But as soon as a select is added, it slows down.